Documentation/v2/admin_guide.md

```diff
@@ -165,13 +165,16 @@ $ ssh 10.0.1.11
 ```
 
 ```sh
-$ kill `pgrep etcd`
+$ systemctl stop etcd2
 ```
 
 #### Copy the data directory of the now-idle member to the new machine
 
+`%data_dir%` is usually `/var/lib/etcd2`.
+
 ```
-$ tar -cvzf infra1.etcd.tar.gz %data_dir%
+$ cd %data_dir%
+$ tar -cvzf infra1.etcd.tar.gz *
```

Comment:
won't untarring this spill the contents of the data directory into the current working directory (instead of creating a data directory in cwd)? Seems like bad form...